src/Zicht/Bundle/PageBundle/Resources/config/services.xml

```diff
@@ -45,6 +46,10 @@
             <argument type="service" id="zicht_url.provider" on-invalid="null" />
             <tag name="zicht_admin.event_propagation" event="zicht_page.view"/>
         </service>
+        <service id="zicht_bundle_page.admin_menu.translate_page_event_propagation_builder" class="%zicht_bundle_page.admin_menu.translate_page_event_propagation_builde.class%">
+            <argument type="service" id="router"/>
```

Comment:
Er is in de klasse een check of de router bestaat, moet hier dan niet een `on-invalid="null"` bij ?

Reply:
En let op typo, die is hier consequent doorgevoerd